shopping_mall/backend/ai/agent/order_graph/nodes.py

```
@@ -0,0 +1,445 @@
+    )
+
+    # ── interrupt: 교환 품목 선택 대기 ────────────────────────────────────
+    user_input = interrupt(prompt)
+
+    if _is_cancel_intent(user_input):
+        return {**state, "abort": True, "response": ORDER_PROMPTS["flow_cancelled"], "is_pending": False}
+
+    # 사용자 입력 파싱 — 번호 기반 또는 "전체"
+    selected = []
+    if "전체" in user_input:
+        for i, oi in enumerate(order_items):
```

Ruff (0.15.10)

[warning] Loop control variable `i` not used within loop body

(B007)